tests/e2e/expected_results/urllib3/urllib3.json

```diff
@@ -1,15 +1,15 @@
 {
     "metadata": {
-        "timestamps": "2022-11-09 20:35:52"
+        "timestamps": "2023-02-15 20:15:49"
     },
     "target": {
         "info": {
             "full_name": "urllib3/urllib3",
             "local_cloned_path": "git_repos/github_com/urllib3/urllib3",
             "remote_path": "https://github.com/urllib3/urllib3",
             "branch": "main",
-            "commit_hash": "87a0ecee6e691fe5ff93cd000c0158deebef763b",
-            "commit_date": "2022-10-04T07:59:23-05:00"
+            "commit_hash": "8619650a81a959b716b2f38b2a714ac1831e9a98",
```

Comment:
This commit hash seems to be the latest commit on the urllib3 repository. In the integration test, we pin the hash of the repository [here](https://github.com/oracle-samples/macaron/blob/07d07ff006ce1230ffe2cb50f7230840e7110a6f/scripts/dev_scripts/integration_tests.sh#L57) so that our results are deterministic. I think you could either update the commit hash of urllib3 in the [integration_test.sh](https://github.com/oracle-samples/macaron/blob/07d07ff006ce1230ffe2cb50f7230840e7110a6f/scripts/dev_scripts/integration_tests.sh#L57) or update this expected result with the analysis on the commit hash specified in [here](https://github.com/oracle-samples/macaron/blob/07d07ff006ce1230ffe2cb50f7230840e7110a6f/scripts/dev_scripts/integration_tests.sh#L57).

Reply:
Updated the expected result with the analysis on the original commit hash, in [7da1e4a](https://github.com/oracle-samples/macaron/pull/67/commits/7da1e4a7a35f4b7d97abe2309fad8fd26780db72)